datafusion/sql/Cargo.toml

```diff
@@ -48,6 +48,7 @@ arrow = { workspace = true }
 bigdecimal = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+datafusion-functions-aggregate = { workspace = true }
```

Comment:
We have tried hard to avoid having the sql parser depend on function libraries -- is there any way to avoid changing this dependency?